apps/aecore/include/blocks.hrl

```diff
@@ -0,0 +1,24 @@
+-record(trees, {}).
+-type(trees() :: #trees{}).
+
+-record(block, {
+          height = 0             :: non_neg_integer(),
+          prev_block_hash = <<>> :: binary(),
+          trees = #trees{}       :: trees(),
```

Comment:
What are these trees? Please add a comment, e.g. _%% Merkle tree of transactions_